RELEASE_NOTES.rst

```diff
@@ -1,4 +1,135 @@
-coala 0.7.0 - baloo
+coala 0.8.0 - grizzly
+=====================
+
+::
+
+          _|\____/|_
+         //  \    /  \
+        //   o    o  |\________________
+       /|            |    \            \
+      | |      []    |     |       \    \
+      |  \   \____/ /       \       |    \
+      |   \________/         "      "     |
+      |                                   |
+      |                  \          \     |
+       \       \         |\          |    |
+        \      /\        | \         |    |
+        |     |  \       |  \        |     \
+        |     |   |     /    |      /\      |
+        |    /    |    |     |     /  |      \
+       /_____}   /_____}    /______}  {_______}
+
+^ Fabian, gimme something better than this :P
+                                                                                                 
+
+For this release, we have had 45 developers from around the world contributing
+over 148 commits in the last 8 weeks.
+
+Improving the API available for bear writers is one of the areas we've focussed
+on for this release, with several new and exciting features. General performance
+has also been improved heavily with some major changes under the hood. The
+documentation has also been worked on, with an emphasis on user-friendliness
+and design.
```

Comment:
I'd add that "major internal changes" have been done to prepare bears for being installed individually in the future